userguide/getting-started/what_is_aviate.adoc

```diff
@@ -0,0 +1,12 @@
+= What Is Aviate?
+
+== About Aviate
+Aviate is designed to deliver a seamless billing experience that builds upon the core capabilities of Kill Bill. Our goal is to provide users with advanced features that cover the entire quote-to-cash lifecycle, including enhanced system observability, streamlined configurations, and valuable business intelligence.
+
+== Beta Program Access
+As part of the Beta program, you'll have access to early feature releases directly on https://aviate.killbill.io[aviate.killbill.io]. This platform will continue to evolve with new functionalities aimed at making billing management easier and more powerful.
+
+== Aviate Plugin
+For Beta customers, we offer the *Aviate plugin*. This plugin extends Kill Bill with additional endpoints specifically for the Beta phase, allowing early access to some of Aviate's backend features. These endpoints are detailed in the *Aviate section* below.
+
+Please refer to the endpoint documentation below to start integrating Aviate into your Kill Bill environment.
```

Comment:
Do we need to add a link to the API docs here?

Reply:
Fixed in [2236cd9](https://github.com/killbill/killbill-docs/pull/571/commits/2236cd9c109b45c48f44d059fcc86d1e689de5e0). I simply referenced the "pages below"


+ Added the missing banner.